docs/getting-started/basics.md

```diff
@@ -0,0 +1,389 @@
+```python exec
+from pcweb.pages.docs import components
+from pcweb.pages.docs.library import library
+from pcweb.pages.docs.custom_components import custom_components
+from pcweb.pages import docs
+import reflex as rx
+```
+
+# Reflex Basics
+
+This page gives an introduction to the most common concepts that you will use to build Reflex apps.
+
+```md section
+# You will learn how to:
+* Create and nest components
+* Customize and style components
+* Distinguishes between compile-time and runtime
+* Display data that changes over time
+* Respond to events and update the screen
+* Render conditions and lists
+* Create pages and navigate between them
+```
+
+[Install]({docs.getting_started.installation.path}) `reflex` using pip.
+
+```bash
+pip install reflex
+```
+
+Import the `reflex` library to get started.
+
+```python
+import reflex as rx
+```
+
+## Creating and nesting components
+
+[Components]({docs.ui.overview.path}) are the building blocks for your app's user interface (UI). They are the visual elements that make up your app, like buttons, text, and images. Reflex has a wide selection of [built-in components]({library.path}) to get you started quickly.
+
+Components are created using functions that return a component object.
+
+```python demo exec
+def my_button():
+    return rx.button("Click Me")
+```
+
+Components can be nested inside each other to create complex UIs. 
+
+To nest components as children, pass them as positional arguments to the parent component. In the example below, the `rx.text` and `my_button` components are children of the `rx.box` component.
+
+```python demo exec
+def my_page():
+    return rx.box(
+        rx.text("This is a page"),
+        # Reference components defined in other functions.
+        my_button()
+    )
+```
+
+You can also use any base HTML element through the `rx.el` namespace.
```

Comment:
we should link to the html elements here https://reflex.dev/docs/library/other/html/

... except i just noticed that this doc is using the Capital letter notation, which is not how the user should use the elements.